spec/dummy/app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Since odds are we'll never ever need CSS and JS in this gem's dummy app, can we remove the entire need for sprockets in the stack? Faster tests, less boilerplate committed.